syncd/Syncd.cpp

```diff
@@ -33,6 +33,10 @@
 #include <iterator>
 #include <algorithm>
 
+#ifdef MDIO_ACCESS_USE_NPU
```

Comment:
im wondering since you are using MDIO_ACCESS_USE_NPU eveyrwhere, and wether it would be better yo remove it and use it once in MdioIpcServer constructor setting context to false, then start and stop thread would not be executed according to your code

Reply:
Changed according to your comment.